src/Libraries/RevitNodes/AnalysisDisplay/PointAnalysisDisplay.cs

```diff
@@ -16,43 +16,6 @@
 
 namespace Revit.AnalysisDisplay
 {
-    [SupressImportIntoVM]
-    [Serializable]
-    public class SpmPrimitiveIdListPair : ISerializable
```

Comment:
for your team to consider how you treat semantic versioning, API compatibility etc, but you might want to consider obsoleting this class instead of deleting it.

I'm assuming you're thinking of treating Dynamo4Revit APIs the same way that Revit treats apis? Meaning it's okay to break them between releases, or after a release where the API is obsolete?

Anyway, something to think about.

Reply:
@mjkkirschner I just found that this class has never been used in code, but I think you are right, I will set it obsolete first.